jorge/js/app.js

```diff
@@ -68,27 +75,50 @@ function login(e) {
   e.preventDefault();
   const loginData = parseFormData(APP_FORMS.LOGIN_FORM);
   const signUpData = JSON.parse(localStorage.getItem(loginData.username));
-
+  userName = loginData.username;
   loginData.balance = signUpData.balance;
 
   allowLogin(loginData, signUpData);
 }
 
+function deposit(e) {
+  e.preventDefault();
+  const depositAmount = parseFormData(APP_FORMS.DEPOSIT_FORM);
+
+  if (depositAmount.deposit !== "") {
+    const userData = JSON.parse(localStorage.getItem(userName));
+    userData.balance += Number(depositAmount.deposit);
+    localStorage.setItem(userName, JSON.stringify(userData));
+    showUserInformation(
+      userData,
+      APP_SECTIONS.MOVEMENTS_WRAPPER,
+      APP_SECTIONS.DEPOSIT_WRAPPER
+    );
+    APP_FORMS.DEPOSIT_FORM.reset();
+  } else {
+    //feedback
+  }
+}
+
 function allowLogin(loginData, signUpData) {
   const isUserRegistered =
     loginData.username === signUpData.username &&
     loginData.password === signUpData.password;
   if (isUserRegistered) {
-    showUserInformation(loginData);
+    showUserInformation(
```

Comment:
I think you can call `toggleVisibility()` and only use this function to assign the text content to each element. Right now `showUserInformation` is having 2 responsibilities, remember a good JS practice principle is the SRP (single responsibility principle)